{{cookiecutter.project_name}}/Applications/{{cookiecutter.app_name}}App/slicer-application-properties.cmake

```diff
@@ -30,7 +30,9 @@ set(WIN_ICON_FILE
   "${CMAKE_CURRENT_LIST_DIR}/Resources/Icons/DesktopIcon.ico"
   )
 
+set(PROJECT_SOURCE_DIR
+  "{{cookiecutter.projection_name}}_SOURCE_DIR"
+  )
 set(LICENSE_FILE
-  "${Slicer_SOURCE_DIR}/License.txt"
+  "${PROJECT_SOURCE_DIR}/License.txt"
```

Comment:
Two updates:

* `License.txt` -> `LICENSE`  (see https://github.com/KitwareMedical/SlicerCustomAppTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/LICENSE)

* Simplify removing the intermediate variable by using the following:

```
${{ '{' }}{{cookiecutter.project_name}}_SOURCE_DIR}/LICENSE
```

The `${{ '{' }}` allows to escape the `{` used to dereference the CMake variable an ensure cookiecutter behave as expected.

Here is an example:

https://github.com/KitwareMedical/SlicerCustomAppTemplate/blob/01336310ee7a600cc1076587e253ba060f8959b4/%7B%7Bcookiecutter.project_name%7D%7D/CMakeLists.txt#L132

See https://github.com/KitwareMedical/SlicerCustomAppTemplate/commit/4e2a3e99984f34b1ccd19e329d14f0816c08ea48

Reply:
Done. That's some funky syntax.